src/app/modules/project/details/details.component.html

```diff
@@ -27,51 +27,45 @@
 <ng-template #projectLoaded>
   <div *ngIf="project; else noProject">
     <div class="row">
-      <div class="col-3 project-metadata">
-        <div class="project-icon" style="background-color: unset">
-          <img [src]="getIconUrl()" alt="Code project" loading="lazy" onerror="this.src='/assets/images/placeholder.svg';">
-        </div>
-        <div *ngIf="project.uri" class="project-metadata__item">
-          <h3>External Link</h3>
-
-          <a href="{{ project.uri }}" rel="noreferrer noopener" target="_blank">{{ project.uri }}</a>
+      <!-- in this place should be a image or a bootstrap carrousel after the wizzard is finished
```

Comment:
English mistakes
"**on this spot** should be **an** image or a bootstrap carrousel after the **wizard** is finished"